### ISSUE TITLE
commit prefix declared in dependabot.yml config file is not used in Pull-Request 

### ISSUE THREAD

**Issue by akumar0212** [tinglesoftware/dependabot-azure-devops#349]
Commit message prefixes defined in the dependabot.yml config file are not being used in Pull-Requests. The .azuredevpops/dependabot.yml config file 

```
version: 2
updates:
  - package-ecosystem: "npm"
    directory: "/"
    schedule:
      interval: "daily"
    commit-message:
      prefix: "chore"
    labels:
      - "skip"
      - "dependencies"
```

Calling it from pipeline task which looks like below
```
- task: dependabot@1
  inputs:
    useConfigFile: true
    gitHubAccessToken: $(PAT)
```

Pull Request should look like
_chore: bump @opentelemetry/instrumentation-express from version1 to version2_

Currently Pull Request looks like
_build(deps): bump @opentelemetry/instrumentation-express from version1 to version2_

**Comment by molszews** [comment]
@mburumaxwell seems like it is not working after #355 got merged

**Comment by molszews** [comment]
yup, other settings are picked up correctly, here is the dependabot.yml:
```
version: 2
updates:
  - package-ecosystem: "npm"
    directory: "/src/app.ui"
    schedule:
      interval: "weekly"
    commit-message:
      prefix: "build(deps)"
      include: "scope"
    open-pull-requests-limit: 1
    milestone: 17166
```
the commit msg I end up with is `Bump @hookform/error-message in /src/app.ui`

**Comment by mburumaxwell** [comment]
@molszews 
In #355, the logic changed from reading an ENV variable to using the config file.
All other settings are extracted in the pipeline task (JavaScript/TypeScript) except `commit-message` and `ignore` which are extracted in the Docker container (Ruby code). For this to work, the configuration file must be in the `.github` folder.

My test seems to work:
File: `.github/dependabot.yml`

```yaml
version: 2
updates:
- package-ecosystem: "nuget"
  directory: "/"
  schedule:
    interval: "weekly"
  commit-message:
    prefix: "build(deps)"
  open-pull-requests-limit: 10
```

My title without the commit-message is: `Bump Tingle.AspNetCore.Authentication from 2.6.0 to 2.7.0` 
After adding the prefix, the title is: `build(deps): bump Tingle.AspNetCore.Authentication from 2.6.0 to 2.7.0`

If you're sure there's a valid problem, kindly open a new issue and provide as much detail as possible.
